src/odemis/acq/move.py

```diff
@@ -365,17 +365,19 @@ def _doCryoSwitchSamplePosition(future, stage, align, target):
             fallback_submoves.append((stage, filter_dict({'x', 'y', 'z'}, target_pos[LOADING])))
             fallback_submoves.append((stage, filter_dict({'rx', 'rz'}, target_pos[LOADING])))
             # Add the sub moves to perform the loading move
-            sub_moves.append((stage, filter_dict({'rx', 'rz'}, target_pos[LOADING])))
             if current_label is UNKNOWN and not stage_referenced:
                 # After referencing the stage could move near the maximum axes range,
                 # and moving single axes may result in an invalid/reachable position error,
                 # so all axes will moved together for this special case.
                 sub_moves.append((stage, filter_dict({'x', 'y', 'z'}, target_pos[LOADING])))
             else:
-                sub_moves.append((stage, filter_dict({'x', 'y'}, target_pos[LOADING])))
+                sub_moves.append((stage, filter_dict({'x'}, target_pos[LOADING])))
```

Comment:
I think this code should have a large comment explaining *why* it's done in this way. What does it solve compared to moving all axes at once, and why this order is better than other order.

If I understand correctly, moving one axis at a time helps because it's easier for the user to notice something is going wrong and pressing the stop (aka "cancel") button. Moving X first is a way to move it to a safe position, as it's not affected by the rx (and rz is typically always 0). Moreover, it's the largest move, and so it'll be "around" the loading position. Every other move is typically much smaller. In case the pivot point is wrong, then z and y might move incorrectly, but it's less likely to cause major damage.